instagram/src/components/CommentSection/Comments.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Comments = props => {
+    return (
+        <div className = 'comments-wrapper'>
+            <h2>{props.comments.username}</h2>
```

Comment:
good clean code great job - could you let me know your process as to how you decided on this code